### PR TITLE
[Slider] Add navigationStep prop

### PR DESCRIPTION
**Pull request by Klynger**
#### What is the purpose of this pull request?

As the title says.

#### What problem is this solving?

Users want to control the amount of slides to pass when you click in the arrows

#### How should this be manually tested?

[workspace](https://sliderprops--storecomponents.myvtex.com/)
You can go to the cms and edit the values too.

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [X] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by iaronaraujo — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -191,6 +191,7 @@ render() {
 | `rootTag` | `string` | :no_entry_sign: | `'div'` | Tag to be rendered in the root of the slider |
 | `sliderFrameTag` | `string` | :no_entry_sign: | `'ul'` | Tag to be rendered in the slider frame element |
 | `threshold` | `number` | :no_entry_sign: | `50` | Minimum of pixels to drag until the slider change the `currentSlide` |
+| `navigationStep`| `number | 'page'` | :no_entry_sign: | `'page'` | How many elements should pass when you click in the arrows to navigate |
```
Comment (emersonlaurentino) [inline docs/README.md]:
```suggestion
| `navigationStep`| `number` | :no_entry_sign: | page | How many elements should pass when you click in the arrows to navigate |
```

File: react/components/Slider.js [hunk 2]
```diff
@@ -487,26 +492,39 @@ class Slider extends PureComponent {
...
     }
 
+    const slidesToSlide = this.getSlidesToSlide()
+
     const arrows = (
       <Fragment>
         {arrowRender({
           orientation: 'left',
-          onClick: scrollByPage ? this.prevPage : () => this.prev(),
+          onClick: () => this.prev(slidesToSlide),
```
Comment (emersonlaurentino) [inline react/components/Slider.js:525]:
```suggestion
          onClick: () => this.prev(this.getSlidesToSlide),
```

File: react/components/Slider.js [hunk 3]
```diff
@@ -487,26 +492,39 @@ class Slider extends PureComponent {
...
         })}
         {arrowRender({
           orientation: 'right',
-          onClick: scrollByPage ? this.nextPage : () => this.next(),
+          onClick: () => this.next(slidesToSlide),
```
Comment (emersonlaurentino) [inline react/components/Slider.js:529]:
```suggestion
          onClick: () => this.next(this.getSlidesToSlide),
```

**Review by emersonlaurentino — CHANGES_REQUESTED** [review]

File: react/components/Slider.js (hunk 2 above)
Comment (Klynger) [reply react/components/Slider.js:525]: If I do this I'll be passing a function as an argument instead of a value

File: docs/README.md (hunk 1 above)
Comment (Klynger) [reply docs/README.md]: I can't remove the `page` option

Comment (emersonlaurentino) [reply docs/README.md]: but I didn't remove it, it already exists in the fourth column, you just added it more than once without need.

File: react/components/Slider.js (hunk 2 above)
Comment (emersonlaurentino) [reply react/components/Slider.js:525]: understand, really.

**Review by emersonlaurentino — APPROVED** [review]

File: react/components/Slider.js [hunk 4]
```diff
@@ -487,26 +492,39 @@ class Slider extends PureComponent {
     }
   }
 
+  getSlidesToSlide = () => {
+    const { navigationStep, scrollByPage } = this.props
+
+    if (typeof navigationStep === 'number') {
+      return navigationStep
+    } else if (navigationStep === constants.NAVIGATION_PAGE || scrollByPage) {
```
Comment (klzns) [inline react/components/Slider.js]: Remove useless else

**Review by victorhmp — APPROVED** [review]

**Review by klzns — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- [Slider] `navigationStep` prop.
```
Comment (klzns) [inline CHANGELOG.md]: It would be nice to write in human form

```suggestion
- `navigationStep` prop to Slider.
```

**Review by klzns — APPROVED** [review]

**Pull request merged**